atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLock.java

```diff
@@ -141,15 +158,16 @@ private long waitForSchemaMutationLock() {
         final long perOperationNodeId = ThreadLocalRandom.current().nextLong(Long.MAX_VALUE - 2);
 
         try {
-            clientPool.runWithRetry((FunctionCheckedException<Cassandra.Client, Void, Exception>) client -> {
+            clientPool.runWithRetry((FunctionCheckedException<Client, Void, Exception>) client -> {
                 Column ourUpdate = lockColumnFromIdAndHeartbeat(perOperationNodeId, 0);
 
                 List<Column> expected = ImmutableList.of(lockColumnWithValue(GLOBAL_DDL_LOCK_CLEARED_VALUE));
 
                 CASResult casResult = writeDdlLockWithCas(client, expected, ourUpdate);
 
                 Column lastSeenColumn = null;
-                int timesAttempted = 0;
+                long lastSeenColumnUpdateTs = 0;
+                int currentTimeBetweenLockAttemptsMillis = TIME_BETWEEN_LOCK_ATTEMPT_ROUNDS_MILLIS;
```

Comment:
Maybe `INITIAL_TIME_BETWEEN_LOCK_ATTEMPT_ROUNDS_MILLIS`?  It's confusing for this to be a final when the actual time between attempts changes.